src/app/common/components/SimpleSelect.tsx

```diff
@@ -19,6 +19,7 @@ export interface ISimpleSelectProps
     SelectProps,
     'onChange' | 'isExpanded' | 'onToggle' | 'onSelect' | 'selections' | 'value'
   > {
+  'aria-label': string; // Make required to satisfy react-axe
```

Comment:
React-axe is enforcing it but shouldn't `aria-label` be mandatory when using user facing fields?

Reply:
Yeah, I only commented to be clear that we're adding it for that reason since it's not clear why PatternFly excludes it. I'm not sure what the deal is with that, I think maybe it's because you can also use `aria-labelledby` technically. I can just remove the comment.